### ISSUE TITLE
Profiles/Progress Card (aka Clinician Dashboard)

### ISSUE THREAD

**Issue by helencarey** [ByunLab/staRt-ionic#363]

**Comment by helencarey** [comment]
@tarabyun @WillHaack 

Hi Tara,
Will and I spoke the Progress Card (aka Clinicians Dashboard) task today.
I've copied your specs over from Asana. 
Even though we have not designed the GUI for this yet, Will could prep the needed for this card.

Is there anything that you would like to amend or change?

**Comment by tarabyun** [comment]
The only thought I have is that I wasn't completely clear when I asked for "Line graph with session number on x-axis, accuracy on y-axis." I think the default interpretation is that this reflects only quest sessions. However, we typically use quizzes to evaluate the baseline the participant is starting from before treatment, and sometimes to track maintenance of gains after treatment. Would it be possible to show quiz as well as quest sessions on the line graph? If so, we would want them to be ordered by date administered (but the x-axis should still display session number, e.g. Quiz 1, Quest 1, rather than date).

**Comment by tarabyun** [comment]
That would also apply to the data table, then--include Quiz as well as Quest sessions. 

Other than that, I think these specs look ready to move to development.

**Comment by WillHaack** [comment]
For the line graph, I don't plan to implement the "total correct" option. The reason is that the line graph will have no meaning if it displays "total correct" since different sessions have a different number of trials.

**Comment by helencarey** [comment]
Yeah, I'd been wondering about this too, while saving highscores. The specs ^ were written a _really_ long time ago, I suspect that at this point Tara was assuming that this table would consist of on-protocol quest sessions only (all 100 trials). @tarabyun, see Will's comment above.

**Comment by WillHaack** [comment]
I consolidated the three headers 'trials scored as correct', 'trials completed' and 'percent correct' to one header: "performance" that displays the information as such:

![image](https://user-images.githubusercontent.com/766792/70576079-94335a00-1b6d-11ea-82c6-bf6a59b3bede.png)

**Comment by tarabyun** [comment]
This is fine with me, thanks!

**Comment by WillHaack** [comment]
My current thoughts: The ticket says to have the option to upload to server and delete from device. I think we should remove/deprioritize this feature since we already have the upload-to-server button on the recordings page. The recordings page is quite similar to the progress dashboard. I even thought of combining them. I didn't because of one major distinction: the progress dashboard does not contain incomplete sessions, while the recording page needs those for the 'resume' button. A separate feature that may be nice is being able to click on a specific session and see the words and their corresponding ratings.

**Comment by tarabyun** [comment]
I'm fine with eliminating the upload/delete recordings feature from the clinician dashboard.

**Comment by tarabyun** [comment]
But @helencarey, feel free to weigh in from a UX perspective
